send-emails-from-a-service/README.md

```diff
@@ -0,0 +1,681 @@
+# Sending emails from a service
+
+## Overview
+
+In this tutorial, you will develop a service that sends an email when a client makes an appointment at a hospital. The email sent to the user includes information about the appointment.
```

Comment:
Atm, this reads like, the service just sends an email when an appointment is made. But, that's not the case, right? It is the reservation service that also sends an email.